From 00215b7facfd4bb46c437411da890575fbce5ba8 Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Fri, 11 Jun 2021 15:41:43 +0200
Subject: [PATCH 319/529] ARM: dts: stm32: add vrefint support to adc2 on
 stm32mp15

Set STM32 ADC2 as a consumer of BSEC, to retrieve
vrefint calibration data on STM32MP15.

Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/283305
Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
Change-Id: I69d7e678616e8c620df6d1428aade6f2f6bce779
---
 arch/arm/boot/dts/stm32mp151.dtsi | 2 ++
 1 file changed, 2 insertions(+)

--- a/arch/arm/boot/dts/stm32mp151.dtsi
+++ b/arch/arm/boot/dts/stm32mp151.dtsi
@@ -1150,6 +1150,8 @@
 				interrupts = <1>;
 				dmas = <&dmamux1 10 0x400 0x01>;
 				dma-names = "rx";
+				nvmem-cells = <&vrefint>;
+				nvmem-cell-names = "vrefint";
 				status = "disabled";
 				channel@13 {
 					reg = <13>;
